src/main/evaluate.c

```diff
@@ -167,6 +167,10 @@ static int cond_do_regex(REQUEST *request, fr_cond_t const *c,
 	rad_assert(lhs_type == PW_TYPE_STRING);
 	rad_assert(lhs != NULL);
 
+	if (lhs_type != PW_TYPE_STRING || lhs == NULL) {
+		return -1;
+	}
+
```

Comment:
The assertions a few lines up should also check for this.  Either Coverity is missing the assertions, OR we should have only one check.  i.e. there's not much point in having assertions *and* run-time checks.